jenkins/debian/debos/scripts/stretchltp.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Important: This script is run under QEMU
+
+set -e
+
+# Build-depends needed to build the test suites, they'll be removed later
+BUILD_DEPS="
+            acl \
+            attr \
+            autoconf \
+            automake \
+            gawk \
+            git \
+            gcc \
+            libaio1 \
+            libc6-dev \
+            libcap2 \
+            make \
+            openssl \
+            zip \
+"
+
+apt-get install --no-install-recommends -y  ${BUILD_DEPS}
+
+BUILDFILE=/build_info.txt
+echo '  "tests_suites": [' >> $BUILDFILE
+
+# Build and install IO performance suites
+########################################################################
+TMP_TEST_DIR="/tmp/tests"
+mkdir -p ${TMP_TEST_DIR} && cd ${TMP_TEST_DIR}
+
+TESTDIR=ltp
+TEST_REPO=https://github.com/linux-test-project/ltp.git
+
+git -c http.sslVerify=false clone --depth=1 ${TEST_REPO}
```

Comment:
You can also add `ca-certificates` in the `BUILD_DEPS` instead of `-c http.sslVerify=false`, it's a bit more secure...